plugins/updater/src/lib.rs

```diff
@@ -3,38 +3,26 @@
 // SPDX-License-Identifier: MIT
 
 use tauri::{
-    plugin::{Builder as PluginBuilder, TauriPlugin},
+    async_runtime::Mutex,
+    plugin::{Builder, TauriPlugin},
     Manager, Runtime,
 };
 
-use tokio::sync::Mutex;
-
 mod commands;
 mod config;
 mod error;
 mod updater;
 
 pub use config::Config;
-pub use error::Error;
+pub use error::{Error, Result};
 pub use updater::*;
-pub type Result<T> = std::result::Result<T, Error>;
-
-struct UpdaterState {
-    target: Option<String>,
-    config: Config,
-}
 
-struct PendingUpdate<R: Runtime>(Mutex<Option<UpdateResponse<R>>>);
-
-#[derive(Default)]
-pub struct Builder {
```

Comment:
We shouldn't remove the Builder here.. installer_args is needed (even the integration test will fail without it IIRC) and a custom target option is nice to support custom configurations (for instance universal macOS bundles, different update channels..)